lib/new-package.js

```diff
@@ -455,11 +457,14 @@ function packageJson(options, packageName) {
       'update-examples': 'node maintenance/update-examples-from-preview.js',
       postinstall: 'elm-tooling install'
     },
-    dependencies: {
+    engines: {
+      node: '>=14.21.3'
+    },
+    devDependencies: {
       'elm-doc-preview': '^5.0.5',
       'elm-review': `^${options.packageJsonVersion}`,
       'elm-test': '^0.19.1-revision10',
-      'elm-tooling': '^1.13.1',
+      'elm-tooling': '^1.15.1',
       'fs-extra': '^9.0.0',
       glob: '^9.3.1',
       'npm-run-all': '^4.1.5'
```

Comment:
Turbo's probably overkill, `npm-run-all2` requires newer node, so I'm leaving as-is. If we find people switching because bugs surface, we can change it, but I think it's fine for now.